airflow/cli/commands/variable_command.py

```diff
@@ -97,6 +107,8 @@ def variables_import(args):
     print(f"{suc_count} of {len(var_json)} variables successfully updated.")
     if fail_count:
         print(f"{fail_count} variable(s) failed to be updated.")
+    if skipped:
+        print(f"The variables with these keys: {list(skipped)} were skipped because they already exists")
```

Comment:
It would be a good idea to sort this for output stability. It would be easier to test as well.